components/google-cloud/google_cloud_pipeline_components/experimental/hyperparameter_tuning_job/component.py

```diff
@@ -0,0 +1,264 @@
+import json
+
+from kfp.components import create_component_from_func_v2
+from typing import NamedTuple
+
+def hyperparameter_tuning_job_run_op(
+    display_name: str,
+    project: str,
+    staging_bucket: str,
+    worker_pool_specs: list,
+    metric_spec: dict,
+    parameter_spec: dict,
```

Comment:
parameter_specs?

Reply:
It is named as [parameter_spec](https://github.com/googleapis/python-aiplatform/blob/master/google/cloud/aiplatform/jobs.py#L1371) in HyperparameterTuningJob, so I'll keep it as it is.

Reply:
Done.